employee_project/main.py

```diff
@@ -2,6 +2,17 @@
 import os
 import time
 from employee import *
+import re
+
+def emp_address():
```

Comment:
Move the method into  a employee.py file, and call this method from main.py
From  main.py try to avoid implementing the core logic  .

Reply:
Changed